### DISCUSSION TITLE
copilot not authorized

### DISCUSSION THREAD

**Discussion by PowaBanga** [community/community#41139] [Copilot Conversations]
### Select Topic Area

Question

### Body

hi everybody,

this is probably only a newbie issue.
i use copilot in vim on my archlinux 
vim-copilot-git r29.2f4f925-2
when i type in vim
`:Copilot setup`
it answer to me
`Copilot: Authenticated as GitHub user PowaBanga`

but copilot seem to not work
if i type this
`Copilot status`
i recieve this answer 
`Copilot: Not authorized`

Any help is welcome, i probably missed something.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @PowaBanga! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by PowaBanga** [comment]
ha yes, you are right, i was lost ^^. thank you for moved my post in the good place :)

**Comment by Akash1134** [comment]
Hi @PowaBanga , Sorry that you're facing trouble using Copilot with Neovim. It seems like there is a problem with connecting to the Copilot server. Please uninstall and then reinstall the plugin, allow it some time to connect, and see if that helps. Also, make sure that you are running the latest version of both Copilot and Neovim. Thanks!
